plugins/obs-outputs/flv-mux.c

```diff
@@ -32,6 +32,85 @@
 #define VIDEODATA_AVCVIDEOPACKET 7.0
 #define AUDIODATA_AAC 10.0
 
+#define VIDEO_FRAMETYPE_OFFSET 4
+enum video_frametype_t {
+	FT_KEY = 1 << VIDEO_FRAMETYPE_OFFSET,
+	FT_INTER = 2 << VIDEO_FRAMETYPE_OFFSET,
+};
+
+// Y2023 spec
+#define FRAME_HEADER_EX (8 << VIDEO_FRAMETYPE_OFFSET)
```

Comment:
Use `const` types as much as possible, avoid using defines (admittedly as C has no `constexpr` there’s
sometimes no other way).

Reply:
Done in the last commit from yuriy-chumak